src/modules/plus/filter_strobe.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * filter_strobe.c -- simple strobing filter
+ * Copyright (C) 2007 Martin Sandsmark <martin.sandsmark@kde.org>
```

Comment:
You should make the copyright year 2020 even though we all hate that year now.

Reply:
haha, but fixed.